components/subsection.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+SubSection.defaultProps = {
+  types: [],
+}
+
+SubSection.propTypes = {
+  types: PropTypes.array,
+}
+
+export default function SubSection({ types, search }) {
+  return (
+    <div class="display-inline-block position-relative float-right small">
+      {types.map((type, key, arr) => {
+        return <div key={key} >
+          <a href="#" onClick={(e) => {
+            e.preventDefault()
+            search(type)
+          }}>{type}</a>
+          {arr.length-1 === key ? '' : ','}
```

Comment:
What about `array.includes(key)` instead?

Reply:
It's checking if it's the last iteration of `map()` and including a comma or not. I think I have a better solution.

Reply:
Gotcha, missed that. Perhaps a simple helper function is in order?